src/ocamlorg_data/ood.mli

```diff
@@ -100,6 +100,25 @@ module Event : sig
   val get_by_slug : string -> t option
 end
 
+module Meetup : sig
+  type location =
```

Comment:
We already have a location type no? We could refactor the module to use the same